src/person/controllers/update_controller.py

```diff
@@ -54,10 +55,11 @@ def process(self):
         self._user.save()
         self._person.save()
 
-        if profile_pic_base64 == "":
+        if not self._profile_pic:
             remove_profile_pic(self._user.id)
-        elif profile_pic_base64 is not None:
-            upload_profile_pic(self._user.id, profile_pic_base64)
+        else:
+            remove_profile_pic(self._user.id)
+            upload_profile_pic(self._user.id, os.environ.get("IMAGE_BUCKET_NAME"), self._profile_pic)
```

Comment:
Make sure when this is deployed that your environment variables are updated on the server if it doesn't include this already.

Reply:
yup, should already be this on the server